### PR TITLE
ci: fix dnt & udd tasks

### PR DESCRIPTION
**Pull request by harrysolovay**

### CODE REVIEW

File: _tasks/dnt.ts [hunk 1]
```diff
@@ -50,5 +52,5 @@ await Promise.all([
     typeCheck: false,
   }),
   fs.copy("LICENSE", path.join(outDir, "LICENSE")),
-  fs.copy("Readme.md", path.join(outDir, "Readme.md")),
+  fs.copy("help.txt", path.join(outDir, "help.txt")),
```
Comment (tjjfvi) [inline _tasks/dnt.ts:55]: We should probably still copy the readme

File: util/branded.ts [hunk 2]
```diff
@@ -1,9 +1,4 @@
...
-
-declare const _hex: unique symbol
-export type Hex = Branded<string, typeof _hex>
-
-declare const _hash: unique symbol
-export type Hash = Branded<Uint8Array, typeof _hash>
-
-export type HexHash = Hex & Hash
+// TODO: delete these upon re-rpc merge
+export type Hex = string
```
Comment (tjjfvi) [inline util/branded.ts:2]: Why this, here?

File: _tasks/dnt.ts (hunk 1 above)
Comment (harrysolovay) [reply _tasks/dnt.ts:55]: If we do, it'll contain dead links (given its in a different directory)

File: util/branded.ts (hunk 2 above)
Comment (harrysolovay) [reply util/branded.ts:2]: DNT started throwing an error regarding serialization of symbols `hex_` and `hash_` (even if we export them from the root mod). This was the fix.

File: _tasks/dnt.ts (hunk 1 above)
Comment (tjjfvi) [reply _tasks/dnt.ts:55]: Better to have a readme with dead links than no readme at all, IMO

Comment (tjjfvi) [inline _tasks/dnt.ts:55]:
```suggestion
  fs.copy("Readme.md", path.join(outDir, "Readme.md")),
  fs.copy("help.txt", path.join(outDir, "help.txt")),
```

**Review by tjjfvi — APPROVED** [review]

**Pull request merged**